clyp/transpiler.py

```
@@ -59,8 +60,50 @@ def _replace_keywords_outside_strings(line: str) -> str:
         parts[i] = part
     return "".join(parts)
 
+def _is_clyp_package(path: str) -> bool:
+    """Check if the given path is inside a Clyp package (has __init__.clyp in its folder or parent folders)."""
+    p = pathlib.Path(path).resolve()
+    if p.is_file():
+        p = p.parent
+    for parent in [p] + list(p.parents):
+        if (parent / "__init__.clyp").exists():
+            return True
+    return False
```

Ruff (0.11.9)

Use `return any((parent / "__init__.clyp").exists() for parent in [p] + list(p.parents))` instead of `for` loop

Replace with `return any((parent / "__init__.clyp").exists() for parent in [p] + list(p.parents))`

(SIM110)